massa-consensus-worker/src/consensus_worker.rs

```diff
@@ -144,18 +144,8 @@ impl ConsensusWorker {
         // notify execution module of current blockclique and final blocks
         // we need to do this because the bootstrap snapshots of the executor vs the consensus may not have been taken in sync
         // because the two modules run concurrently and out of sync
-        channels.execution_controller.update_blockclique_status(
-            block_db.clone_all_final_blocks(),
-            block_db
-                .get_blockclique()
-                .into_iter()
-                .filter_map(|block_id| {
-                    block_db
-                        .get_active_block(&block_id)
-                        .map(|a_block| (block_id, a_block.block.clone()))
-                })
-                .collect(),
-        );
+
+        // TODO: use shared-storage.
```

Comment:
If we want to merge this is that a problem that this is still a todo and the past behaviour has been removed ? Or it's ok because you store here : https://github.com/massalabs/massa/pull/2233/files#diff-990c58de6c3e1194da0ee5db7bcac14cdd0816ff8682805753c594f83e304511R589 ?

Reply:
Good observation. For now we'll keep this branch open, and makes changes to it, as well as try to merge the other PRs into it. 

This particular TODO is addressed in https://github.com/massalabs/massa/pull/2322